dpctl/tensor/_elementwise_funcs.py

```diff
@@ -264,6 +266,8 @@
 Return:
     usm_ndarray:
         An array containing the element-wise exp(x)-1 values.
+        The data type of the returned array is determined by the Type
+        Promotion Rules..
```

Comment:
```suggestion
        Promotion Rules.
```

Reply:
This comment has been resolved, but is there a reason not to fix the typo?

Reply:
sorry, I made a mistake. Nikita had fixed it and I just updated my local branch with main branch. I forgot to update the local branch first to include Nikita's changes. I will fix it right now.